ui/app/services/capabilities.ts

```diff
@@ -31,12 +40,30 @@ export default class CapabilitiesService extends Service {
     return assert('query object must contain "paths" or "path" key', false);
   }
 
-  /*
-  this method returns a capabilities model for each path in the array of paths
-  */
-  async fetchMultiplePaths(paths: string[]): Promise<Array<CapabilitiesModel>> | AdapterError {
+  async fetchMultiplePaths(paths: string[]): MultipleCapabilities | AdapterError {
```

Comment:
```suggestion
  async fetchMultiplePaths(paths: string[]): Promise<MultipleCapabilities | AdapterError> {
```

Reply:
This actually doesn't return a promise! On line 45 we await the request and this just returns an object with the path as the key and value as the capabilities:

```js
{ 
 "my/api/path": {
     canCreate: true,
     canDelete: true,
     canList: true,
     canPatch: true,
     canRead: true,
     canSudo: true,
     canUpdate: true,
   };
}
```

Reply:
Which reminds me that I need to remove the try...catch wrapping the whole thing!

Reply:
I thought any async method technically returns a promise... 🤔

Reply:
async is just necessary so we can await the promise on line 45, the method itself is not a promise, though!